src/reducers/map-style-updaters.js

```diff
@@ -213,7 +215,8 @@ function getLayerGroupsFromStyle(style) {
 export const initMapStyleUpdater = (state, action) => ({
   ...state,
   // save mapbox access token to map style state
-  mapboxApiAccessToken: (action.payload || {}).mapboxApiAccessToken
+  mapboxApiAccessToken: (action.payload || {}).mapboxApiAccessToken,
+  mapboxApiUrl: (action.payload || {}).mapboxApiUrl
```

Comment:
you can discard all default mapStyles here when `mapStylesReplaceDefault=true`
```js
export const initMapStyleUpdater = (state, action) => ({
  ...state,
  // save mapbox access token to map style state
  mapboxApiAccessToken: (action.payload || {}).mapboxApiAccessToken,
  mapboxApiUrl: (action.payload || {}).mapboxApiUrl,
  mapStyles: (action.payload || {}).mapStylesReplaceDefault ? {} : state.mapStyles
});
```

Reply:
🍏 Done